jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/URIUtil.java

```diff
@@ -1890,6 +1898,7 @@ public static URI toURI(String resource)
                     // Input is a possible Windows path disguised as a URI "D:/path/to/resource.txt".
                     try
                     {
+                        // TODO remove this method or at least move this aspect of it to a ResourceFactory
```

Comment:
I think this is safe if we can prove the path is absolute.

Reply:
I just don't think it is a necessary method.   see above.

Reply:
Eliminating this method also introduces more demands on ResourceFactory and tracking of mounts.

Reply:
I've always been sure that it is Resources job to work out if a string is a URI or a Path or something to be mounted/unpacked.  The URIUtil.toURI method was only introduced recently and for some reason it took over the Path logic.
I'm very comfortable removing this method... in fact I've done it in see #11577